spec/fixtures/reserved_keywords.yml

```diff
@@ -1,2 +1,4 @@
 select: 123
 collect: 456
+section:
```

Comment:
Why adding this keyword deeper in the tree instead of on the same level as other reserved keywords?